bull-transformer-generator/transformer-generator-core/src/main/java/com/hotels/beans/generator/core/Transformer.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (C) 2019-2020 Expedia, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.beans.generator.core;
+
+/**
+ * A {@code Transformer} can convert an object of a source type to a destination type.
+ * @param <A> the source type
+ * @param <B> the destination type
+ * @author mmirk
```

Comment:
The author is not allowed by the Hotels.com OSS rules, could you please remove them from all files? thanks

Reply:
Thanks, will do that!